implementations/rust/ockam/ockam_command/src/cmd/identity.rs

```diff
@@ -1,33 +1,31 @@
-use crate::{args::IdentityOpts, identity::save_identity, storage};
+use crate::{args::IdentityOpts, identity::save_identity, storage, OckamVault};
 use anyhow::Context as Ctx;
-use ockam::{identity::*, vault::*, Context};
+use ockam::{identity::*, Context};
+use ockam_vault::storage::FileStorage;
+use std::sync::Arc;
 
 pub async fn run(args: IdentityOpts, mut ctx: Context) -> anyhow::Result<()> {
     let ockam_dir = storage::init_ockam_dir()?;
     let id_path = ockam_dir.join("identity.json");
-    let vault_path = ockam_dir.join("vault.json");
-    if id_path.exists() || vault_path.exists() {
+    if id_path.exists() {
         if !args.overwrite {
             anyhow::bail!(
                 "An identity or vault already exists in {:?}. Pass `--overwrite` to continue anyway",
                 ockam_dir
             );
         }
-        if vault_path.exists() {
-            std::fs::remove_file(&vault_path)
-                .with_context(|| format!("Failed to remove {:?}", vault_path))?;
-        }
```

Comment:
Is this directory cleared somewhere else or was this functionality just removed?

Reply:
We now have Vault Storage that is responsible for that